gradle/libs.versions.toml

```diff
@@ -301,6 +301,16 @@ ktor = [
     "ktorGson"
 ]
 
+traceCore = [
+    "moshi",
+    "jctools",
+    "kotlin",
```

Comment:
probably this one can be removed from the bundle

Reply:
which one ? `kotlin` ? I think yes you're right.